zhaquirks/osram/a19twhite.py

```diff
@@ -30,6 +37,7 @@ class A19TunableWhite(CustomDevice):
         # <SimpleDescriptor endpoint=3 profile=260 device_type=258
         # device_version=2 input_clusters=[0, 3, 4, 5, 6, 8, 768, 64527]
         # output_clusters=[25]>
+        MODELS_INFO: [(OSRAM, "LIGHTIFY A19 Tunable White")],
```

Comment:
```suggestion
        MODELS_INFO: [(OSRAM, "LIGHTIFY A19 Tunable White"), (OSRAM, "LIGHTIFY RT Tunable White")],
```

Reply:
the OSRAM recessed lighting has exactly the same signature, just different model. Lets add this model too and maybe rename the quirk class to slightly more neutral, eg just `OsramTunableWhite`? Don't really bother me, just recessed lights are not A19 types :)